spi/identity-hub-spi/src/main/java/org/eclipse/dataspaceconnector/identityhub/credentials/VerifiableCredentialsJwtService.java

```diff
@@ -45,4 +50,26 @@ public interface VerifiableCredentialsJwtService {
      * @return VerifiableCredential represented as {@code Map.Entry<String, Object>}.
      */
     Result<Map.Entry<String, Object>> extractCredential(SignedJWT jwt);
+
+    /**
+     * Builds a verifiable credential as a signed JWT
+     *
+     * @param credential The verifiable credential to sign
+     * @param issuer The issuer of the verifiable credential
+     * @param subject The subject of the verifiable credential
+     * @param jwk The private key of the issuer, used for signing
+     *
+     * @return The Verifiable Credential as a JWT
+     * @throws Exception In case the credential can not be signed
+     */
+    SignedJWT buildSignedJwt(VerifiableCredential credential, String issuer, String subject, ECKey jwk) throws Exception;
```

Comment:
How about using `PrivateKeyWrapper` here in order to avoid binding a specific key type and reuse EDC library code?